converter.py

```diff
@@ -58,7 +59,11 @@ def print_help():
         dest_dir = args[2]
         if not os.path.exists(src_path):
             print('Given db file path is incorrect. This filepath doesn\'t exists ::' + src_path)
-        file_dir = os.path.dirname(dest_dir)
+        case_name = src_path.split("/")[-3]
+        folder_name = os.path.splitext(os.path.basename(src_path))[0]
+        file_dir = dest_dir+"/"+case_name+"/"+folder_name
+        if os.path.exists(file_dir):
```

Comment:
I think if `dir` is existing then instead of removing and creating new one, better we keep it. So basically just check if it exists or not. If not exist then create, else if exist then move a head

Reply:
Okay!